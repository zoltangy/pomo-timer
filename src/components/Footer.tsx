import React from "react";
import { Divider, Grid } from "@material-ui/core";
import { makeStyles } from "@material-ui/core/styles";
import signature from "../assets/sign.png";

const useStyles = makeStyles((theme) => ({
  root: {
    paddingBottom: theme.spacing(2),
    position: "absolute",
    bottom: 0,
    width: "100%",
  },
  signature: {
    marginTop: theme.spacing(2),
  },
}));

export default function Footer() {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <Grid container justify="center" spacing={0}>
        <Grid item xs={12}>
          <Divider variant="middle" />
        </Grid>
        <Grid item xs={12} container justify="center" className={classes.signature}>
          <img src={signature} alt="signature" width="50" height="44" />
        </Grid>
      </Grid>
    </div>
  );
}
