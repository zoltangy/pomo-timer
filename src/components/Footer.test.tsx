import React from "react";
import { render } from "@testing-library/react";
import Footer from "./Footer";

describe("Footer tests", () => {
  test("that snapshots match", () => {
    const { asFragment } = render(<Footer />);

    expect(asFragment()).toMatchSnapshot();
  });
});
