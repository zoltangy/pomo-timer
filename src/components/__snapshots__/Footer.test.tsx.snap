// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests that snapshots match 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-1"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-justify-xs-center"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <hr
          class="MuiDivider-root MuiDivider-middle"
        />
      </div>
      <div
        class="MuiGrid-root makeStyles-signature-2 MuiGrid-container MuiGrid-item MuiGrid-justify-xs-center MuiGrid-grid-xs-12"
      >
        <img
          alt="signature"
          height="44"
          src="sign.png"
          width="50"
        />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
